.github/workflows/test-macos.yml

```
@@ -58,6 +58,6 @@ jobs:
       - name: tools/doc/node_modules workaround
         run: make tools/doc/node_modules
       - name: Build
-        run: make build-ci -j3 V=1 CONFIG_FLAGS="--error-on-warn"
+        run: make build-ci -j`getconf _NPROCESSORS_ONLN` V=1 CONFIG_FLAGS="--error-on-warn"
```

```suggestion
        run: make build-ci -j$(getconf _NPROCESSORS_ONLN) V=1 CONFIG_FLAGS="--error-on-warn"
```
Prefer `$()` for command substitution instead of \`\` since the backticks are old-style and deprecated.